src/pages/index.js

```diff
@@ -15,60 +15,66 @@ import Services from '../components/Homepage/services'
 import GreyBackground from '../components/GreyBG'
 
 const IndexPage = ({
-  data: { contentfulHomepage: content, allContentfulMeetupEvent: events, site }
-}) => (
-  <Layout>
-    <Helmet
-      title={`${site.siteMetadata.title} ${
-        content.seoTitle ? '- ' + content.seoTitle : ''
-      } `}
-      meta={[{ name: 'description', content: content.seoMetaDescription }]}
-    >
-      <html lang="en" />
-    </Helmet>
-    <Grid>
-      <Padding bottom={{ smallPhone: 0, smallTablet: 2, desktop: 2 }}>
-        <CaseStudy
-          caseStudy={content.featuredCaseStudy}
-          subHeading="Featured work"
-        />
-      </Padding>
-      <Padding bottom={{ smallPhone: 0, desktop: 2 }} />
-    </Grid>
-    <GreyBackground topMargin>
+  data: { contentfulHomepage: content, allContentfulMeetupEvent: events, site },
+  location
+}) => {
+  return (
+    <Layout location={location}>
+      <Helmet
+        title={`${site.siteMetadata.title} ${
+          content.seoTitle ? '- ' + content.seoTitle : ''
+        } `}
+        meta={[{ name: 'description', content: content.seoMetaDescription }]}
+      >
+        <html lang="en" />
+      </Helmet>
       <Grid>
-        <Padding top={{ smallPhone: 2 }} />
-        <Padding top={{ smallPhone: 4, smallTablet: 5, desktop: 6 }} bottom={3}>
-          <SEOText text={content.seoText.content[0].content} />
-          <Padding bottom={{ smallPhone: 2, smallTablet: 4, desktop: 4 }} />
-          <Companies companies={content.companies} />
+        <Padding bottom={{ smallPhone: 0, smallTablet: 2, desktop: 2 }}>
+          <CaseStudy
```

Comment:
A minor thing, but it might be good to mention changes like this in the PR description? Just so a reviewer knows to check whether anything has changed unintentionally